components/button/stories/template.js

```diff
@@ -33,6 +33,38 @@ export const Template = ({
   }
 
   return html`
+    <button
```

Comment:
I'm torn because I love this view for the storybook but I also know how helpful it is to keep the template.js files isolated to rendering a single component (for re-use elsewhere). Maybe we can keep the template scoped to 1 button but update the button.stories.js to render a custom template calling it 4x?

<img width="332" alt="Screenshot 2023-04-10 at 9 10 33 AM" src="https://user-images.githubusercontent.com/1840295/230907518-28b5e5c8-0059-429e-a596-52131a214548.png">